resources/zib/zib-SNAQScore.xml

```diff
@@ -0,0 +1,225 @@
+  <purpose value="This Observation resource represents the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) SNAQScore-v3.2(2020EN)](https://zibs.nl/wiki/SNAQScore-v3.2(2020EN))." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-snaqscore-v3.2-2020EN" />
+    <uri value="https://zibs.nl/wiki/SNAQScore-v3.2(2020EN)" />
+    <name value="zib SNAQScore-v3.2(2020EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="true" />
+  <type value="Observation" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Observation" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Observation">
+      <path value="Observation" />
+      <short value="SNAQScore" />
+      <alias value="SNAQScore" />
+      <mapping>
+        <identity value="zib-snaqscore-v3.2-2020EN" />
+        <map value="NL-CM:4.6.1" />
+        <comment value="SNAQScore" />
+      </mapping>
+    </element>
+    <element id="Observation.code">
+      <path value="Observation.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://snomed.info/sct" />
+          <code value="8881000146104" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Observation.effective[x]">
+      <path value="Observation.effective[x]" />
+      <slicing>
+        <discriminator>
+          <type value="type" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.effective[x]:effectiveDateTime">
+      <path value="Observation.effectiveDateTime" />
+      <sliceName value="effectiveDateTime" />
+      <short value="SNAQScoreDateTime" />
+      <definition value="The date and time at which the SNAQ score was determined." />
+      <alias value="SNAQScoreDatumTijd" />
+      <type>
+        <code value="dateTime" />
+      </type>
+      <mapping>
+        <identity value="zib-snaqscore-v3.2-2020EN" />
+        <map value="NL-CM:4.6.6" />
+        <comment value="SNAQScoreDateTime" />
+      </mapping>
+    </element>
+    <element id="Observation.value[x]">
+      <path value="Observation.value[x]" />
+      <slicing>
+        <discriminator>
+          <type value="type" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.value[x]:valueInteger">
+      <path value="Observation.valueInteger" />
```

Comment:
Zou dit path niet `Observation.value[x]` moeten zijn?

Reply:
I agree; this should be generated automatically in Forge right?

Reply:
Yes. I experienced that Forge corrects this after a resave.

Reply:
Apparently there was a bug in Forge, since this same issue was also mentioned in the Forge Support inbox (but it's resolved now).